projects/vdk-heartbeat/src/vdk/internal/heartbeat/vdk_heartbeat_data_job/database_ingestion/20_verify_data.py

```diff
@@ -0,0 +1,45 @@
+# Copyright 2021 VMware, Inc.
+# SPDX-License-Identifier: Apache-2.0
+import logging
+import time
+
+from vdk.api.job_input import IJobInput
+
+log = logging.getLogger(__name__)
+
+
+def run(job_input: IJobInput):
+    # If db property includes catalog name, get only the schema name ('default' from 'memory.default').
+    # This is done because the templates' implementation escapes db, table and column names passed as arguments, so that
+    # reserved words could be used for their values
+    db_name = job_input.get_property("db").split(".")[-1]
+
+    props = job_input.get_all_properties()
+    destination_table = props["destination_table"]
+    id1 = props["id1"]
+    id2 = props["id2"]
+    id3 = props["id3"]
+
+    timeout = 300  # [seconds]
```

Comment:
timeout should be configurable. It's basically the first thing people need to configure in their test environment.

Reply:
Agreed